charts/ocis/templates/_common/customcachain.tpl

```diff
@@ -0,0 +1,25 @@
+{{/*
+oCIS ca mount
+
+*/}}
+{{- define "ocis.caPath" -}}
+{{- if .Values.customCAChain.enabled }}
+- name: custom-ca-chain
+  mountPath: /etc/ssl/custom/
+{{- end }}
+{{- end -}}
+
+{{- define "ocis.caVolume" -}}
+{{- if .Values.customCAChain.enabled }}
+- name: custom-ca-chain
+  configMap:
+    name: {{ .Values.customCAChain.existingConfigMap }}
```

Comment:
I see a glitch here. Only `enabled` is check but the `existingConfigMap` is used as-is. This might be empty which will lead to issues. So both should be checked and/or an error should be presented.

Reply:
actually if `enabled` is set to `true`, we should use `fail` if `existingConfigMap` == ""

Reply:
I used require ... it seems more appropriate